.vscode/tasks.json

```diff
@@ -8,9 +8,10 @@
       "type": "docker-compose",
       "dockerCompose": {
         "up": {
+          "profile": "local",
           "detached": true,
           "build": true,
-          "services": ["cosmos", "mock-api", "oidc-server"]
+          "services": ["cosmos", "mock-api", "oidc-server", "next-app", "api"]
```

Comment:
Do you need to specify services if you have profile? Could you omit one of these as they are both ways of specifying which services to bring up

Reply:
Removed services now we are using the profile.